pastas/stats/tests.py

```diff
@@ -411,8 +415,8 @@ def stoffer_toloi(series, lags=15, nparam=0, freq="D",
     return qm, pval
 
 
-def diagnostics(series, alpha=0.05, nparam=0, lags=15, stats=(),
-                float_fmt="{0:.2f}"):
+def diagnostics(series: Series, alpha: float = 0.05, nparam: int = 0, lags: int = 15, stats: tuple = (),
```

Comment:
Tuple with capital T?

Reply:
Not needed here because the type is `tuple`. `Tuple` with capital T is only needed for Python version 3.8 and lower if a tuple would be called like: `tuple[...]`. Same goes for `list` and `List`.